backends/mediatek/scripts/README.md

```diff
@@ -1,13 +1,35 @@
-# Build Step
-1. Download **Buck2**
-Download buck2 from https://github.com/facebook/buck2/releases/tag/2023-07-18, and add the directory to $PATH
+# Build Instructions
 
-2. Download **Android NDK**
-Download Android NDK and make sure that $ANDROID_NDK is specified as the NDK Path.
+This document provides a step-by-step guide to set up the build environment for the MediaTek ExercuTorch libraries.
 
-3. Download **MediaTek ExercuTorch libraries**
-Download related libraries.
+## Prerequisites
 
-4. Follow the ExercuTorch official tutorial [Setting up executorch](https://pytorch.org/executorch/stable/getting-started-setup)
+Before you begin, ensure you have the following prerequisites installed and configured:
 
-5. Run **mtk_build.sh**
\ No newline at end of file
+### 1. Buck2 Build Tool
+
+- **Download Buck2**: Obtain Buck2 from the official [releases page](https://github.com/facebook/buck2/releases/tag/2023-07-18).
+- **Add to PATH**: Extract the downloaded file and add the directory to your system's `$PATH` environment variable.
+
+### 2. Android NDK
+
+- **Download Android NDK**: Acquire the Android NDK from the [Android developer site](https://developer.android.com/ndk/downloads).
+- **Set NDK Path**: Ensure that the `$ANDROID_NDK` environment variable is set to the path where the NDK is located.
+
+### 3. MediaTek ExercuTorch Libraries
+
+Download the following libraries required for MediaTek ExercuTorch:
```

Comment:
> Download the following libraries required ...

rephrase it to "Download the following libraries required for MediaTek libraries from xx portal (linked to be added)" just so it's clearer where it comes from